templates/landscape/install.html

```diff
@@ -182,183 +157,7 @@ <h2>Install Landscape Server to manage machines and containers</h2>
       </div>
       <hr class="is-fixed-width">
 
-      <div class="p-tabs__content" tabindex="0" role="tabpanel" id="automated-tab" aria-labelledby="automated" style="scroll-margin-top:86px;">
-        <div class="row">
-          <div class="col-8 col-start-large-4 p-strip is-shallow">
-            <p>The entire installation and configuration can be automated with <a href="https://cloud-init.io/">Cloud Init</a>, using this <a href="https://github.com/canonical/landscape-scripts/blob/main/provisioning/cloud-init.yaml">cloud-init.yaml</a> file.</p>
```

Comment:
I am told "Cloud Init" should be written as "Cloud-init" here. I have made this correction in the copydoc as well.